.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -105,14 +204,28 @@ jobs:
         with:
           workflow: .github/workflows/rawls-swat-tests.yaml
           repo: broadinstitute/terra-github-workflows
-          ref: refs/heads/main
+          ref: refs/heads/iv-az-e2e-1
+          token: ${{ env.TOKEN }}
+          inputs: '{ "bee-name": "${{ env.BEE_NAME }}", "ENV": "qa", "ref": "refs/heads/${{ needs.init-github-context.outputs.branch }}", "test-group-name": "workspaces_azure", "test-command": "${{ env.rawls_test_command }}", "java-version": "17", "billing-project": "${{ needs.az-e2e-params-gen.outputs.project-name }}", "e2e": "true", "service-account": "${{ inputs.service-account }}" }'
```

Comment:
not an issue for PoC but java verions might not want to be hardcoded.

Reply:
Probably can leave it up to `Rawls` team to decide how they would like to approach this.